spki/tests/spki.rs

```diff
@@ -73,6 +73,7 @@ fn decode_and_fingerprint_spki() {
 
 #[test]
 #[cfg(feature = "pem")]
+#[cfg(feature = "fingerprint")]
```

Comment:
Normally we put these on a single line ala:

```suggestion
#[cfg(all(feature = "pem", feature = "fingerprint"))]
```